fdbclient/DatabaseConfiguration.cpp

```diff
@@ -397,11 +397,10 @@ std::string DatabaseConfiguration::configureStringFromJSON(const StatusObject& j
 			continue;
 		}
 
```

Comment:
The fix you provide is too complex. Seems adding a check here that result is not empty would be a nicer solution.